dataland-batch-manager/src/main/kotlin/org/dataland/datalandbatchmanager/service/CompanyUploader.kt

```diff
@@ -126,4 +127,33 @@ class CompanyUploader(
             )
         }
     }
+
+    /**
+     * Updates the ISINs of all companies.
+     * @param leiIsinMap the delta-map with the format "LEI"->"ISIN1,ISIN2,..."
+     */
+    fun updateIsinMapping(leiIsinMap: Map<String, String>) {
```

Comment:
I like `isinMapping`, however, we should really stay consistent with the naming! The parameter is called `leiIsinMap` I saw `gleifMapping` somewhere, I think, and only `mapping` as well e.g. in the application.properties.

Reply:
done